fvm/context.go

```diff
@@ -19,11 +19,13 @@ type Context struct {
 	ServiceAccountEnabled            bool
 	RestrictedAccountCreationEnabled bool
 	RestrictedDeploymentEnabled      bool
+	LimitAccountStorage              bool
 	CadenceLoggingEnabled            bool
 	SetValueHandler                  SetValueHandler
 	SignatureVerifier                SignatureVerifier
 	TransactionProcessors            []TransactionProcessor
 	ScriptProcessors                 []ScriptProcessor
+	StorageCapacityResolver          StorageCapacityResolver
```

Comment:
I'm not 100% sold that Resolver should be part of the context, I think it might actually be better to be part of the env.

Reply:
As per discussion This will be addressed in a separate PR.